src/components/Navbar/NavbarBody/index.tsx

```
@@ -20,10 +20,12 @@ import Spinner from '@/dls/Spinner/Spinner';
 import IconMenu from '@/icons/menu.svg';
 import IconSearch from '@/icons/search.svg';
 import {
-  setIsSearchDrawerOpen,
-  setIsNavigationDrawerOpen,
+  selectIsNavigationDrawerOpen,
   setDisableSearchDrawerTransition,
+  setIsNavigationDrawerOpen,
+  setIsSearchDrawerOpen,
 } from '@/redux/slices/navbar';
+import { isLoggedIn } from '@/utils/auth/login';
```

ESLint

[error] `@/utils/auth/login` import should occur after import of `@/redux/slices/QuranReader/sidebarNavigation`

(import/order)